Tests/Functional/Common/SolrSearchTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * (c) Kitodo. Key to digital objects e.V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo and TYPO3 projects.
+ *
+ * @license GNU General Public License version 3 or later.
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ */
+
+namespace Kitodo\Dlf\Tests\Functional\Common;
+
+use Kitodo\Dlf\Common\Solr\Solr;
+use Kitodo\Dlf\Common\Solr\SolrSearch;
+use Kitodo\Dlf\Domain\Repository\DocumentRepository;
+use Kitodo\Dlf\Domain\Repository\SolrCoreRepository;
+use Kitodo\Dlf\Tests\Functional\FunctionalTestCase;
+use TYPO3\CMS\Extbase\Persistence\Generic\PersistenceManager;
+
+class SolrSearchTest extends FunctionalTestCase
+{
+    private static array $databaseFixtures = [
+        __DIR__ . '/../../Fixtures/Common/solrcores.csv'
+    ];
+
+    private static array $solrFixtures = [
+        __DIR__ . '/../../Fixtures/Common/documents_1.solr.json'
+    ];
+
+    private Solr $solr;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->setUpData(self::$databaseFixtures);
+        $this->solr = $this->setUpSolr(4, 0, self::$solrFixtures);
+    }
+
+    /**
+     * @test
+     */
+    public function canPrepareAndSubmit()
+    {
+        $this->markTestSkipped('Does not work in combination with other tests.');
```

Comment:
Hm, a skipped test is quite useless... There should be a way to make this work even in combination with the other tests. Where exactly does the test fail?